pages/learn/manage/billing.md

```diff
@@ -44,27 +44,43 @@ Further down, you'll find a place to download invoices, add or edit payment meth
 
 At the bottom of this page, you'll find more information on the available [plans][plans], and you can change plans as necessary.
 
+## Credits
+
+To enable customers to drive down their device usage costs, balena has enabled a system known as "Credits". A Credit is a pre-purchased unit that corresponds to the usage of one device exceeding plan allowance for one month. Any unused Credits would be used before a dynamic charge is initiated at the end of your monthly billing cycle.
+
+The discount premise is simple—the greater your volume in a single purchase, the lower your per-Credit cost will be. The discount computation also takes into account the previously purchased and unused Credits. Credit purchases of any size will result in paying less for device usage on the platform. To estimate Credit pricing at different volumes, head over to the "Volume Device Purchase Calculator" section on the [pricing page](https://www.balena.io/pricing/#devices).
+
+This system is available to all non-Prototype customers. Our Pilot, Production, and Enterprise customers have full access to Credit discounts by paying in advance for device usage for any volume.
+
+To manage Credits for a specific organization, navigate to the organization and select the `Credits` tab from the sidebar on the left:
+
+<img src="/img/common/billing/credits_nav.png" width="25%">
+
+On the Credits page, you're able to see Credit purchase history and the number of Credits that have been purchased and are still allocatable for device usage. This is also where you can initiate Credit purchases. An invoice will be automatically generated and listed on the Billing page when a bulk Credit purchase is initiated.
+
+Credits will expire 10 years after the date of purchase. If there is no activity for 2 years we may at our discretion cancel any outstanding credits and will provide 60 days notice to the customer of our intention to do so. Credit purchases are non-refundable and cannot be refunded due to project or plan cancellation.
```

Comment:
Are credits transferable?

Reply:
They are not at the moment, maybe between orgs later. Good catch, I'll add this now @vipulgupta2048

Reply:
Yeah, it's what my mind asked? So I thought I drop it in even if it is nitpick. Solid PR overall, approved it.